_publications/jors/paper.tex

```diff
@@ -98,91 +96,171 @@ \section*{Introduction}
+
+\begin{table}[h!]
+  \centering
+  \begin{tabular}{|l|l|}
+    \hline
+    Language         & Version \\
+    \hline
+    HTML             & 5       \\
+    Jupyter Notebook & 5.4.0   \\
+    JavaScript       & 5       \\
+    Nix              & 2.1.3   \\
+    CoffeeScript     & 1.12.7  \\
+    CSS              & 4       \\
+    \hline
+  \end{tabular}
+  \caption{PFHub programming languages and corresponding supported
+    versions.}
+  \label{tab:versions}
+\end{table}
+
 
 \section*{Additional system requirements}
 
-\textcolor{blue}{E.g. memory, disk space, processor, input devices, output devices.}
+There are no additionial system requirements.
 
 \section*{Dependencies}
 
-\textcolor{blue}{E.g. libraries, frameworks, incl. minimum version compatibility.}
+The entire environment can be built using Nix so the only required
+dependency is a functional Nix installation.
 
 \section*{List of contributors}
 
-\textcolor{blue}{Please list anyone who helped to create the software (who may also not be an author of this paper), including their roles and affiliations.}
+1. Wheeler, Daniel; A, \githublink{wd15} \\
+2. Keller, Trevor; A, \githublink{tkphd} \\
+3. DeWitt, Stephen J.; E, \githublink{stvdwtt} \\
+4. Jokisaari, Andrea M.; D, \githublink{amjokisaari} \\
+5. Schwen, Daniel; D, \githublink{dschwen} \\
+6. Guyer, Jonathan E.; A, \githublink{guyer} \\
+
+See the contributors list on GitHub~\cite{contributors}.
 
 \section*{Software location:}
 
-{\bf Archive} \textcolor{blue}{(e.g. institutional repository, general repository) (required – please see instructions on journal website for depositing archive copy of software in a suitable repository)}
+{\bf Archive}
 
 \begin{description}[noitemsep,topsep=0pt]
-	\item[Name:] \textcolor{blue}{The name of the archive.}
-	\item[Persistent identifier:] \textcolor{blue}{e.g. DOI, handle, PURL, etc.}
-	\item[Licence:] \textcolor{blue}{Open license under which the software is licensed.}
-	\item[Publisher:]  \textcolor{blue}{Name of the person who deposited the software.}
-	\item[Version published:] \textcolor{blue}{The version number of the software archived.}
-	\item[Date published:] \textcolor{blue}{dd/mm/yy}
+	\item[Name:] Zenodo
+	\item[Persistent identifier:] 10.5281/zenodo.2592705
+	\item[Licence:] NIST Public Domain License~\cite{nistlicense}
+	\item[Publisher:]  Daniel Wheeler
+	\item[Version published:] v0.1
+	\item[Date published:] 13/03/19
 \end{description}
 
 
-
-{\bf Code repository} \textcolor{blue}{(e.g. SourceForge, GitHub etc.) (required)}
+{\bf Code repository}
 
 \begin{description}[noitemsep,topsep=0pt]
-	\item[Name:] \textcolor{blue}{The name of the archive.}
-	\item[Persistent identifier:] \textcolor{blue}{e.g. DOI, handle, PURL, etc.}
-	\item[Licence:] \textcolor{blue}{Open license under which the software is licensed.}
-	\item[Date published:] \textcolor{blue}{dd/mm/yy}
+	\item[Name:] GitHub
+	\item[Persistent identifier:] 10.5281/zenodo.2592705
```

Comment:
In the example paper, I believe this is the GitHub URL, since the Zenodo DOI won't take you there.